GeneralizedMorphismsForCAP/gap/SerreQuotients.gi

```diff
@@ -63,3 +63,33 @@ InstallMethod( LiftNaturalIsoFromIdToSomeToSerreQuotientCategory,
     return new_nat_trans;
     
 end );
+
+BindGlobal( "CAP_INTERNAL_INSTALL_OPERATIONS_FOR_SERRE_QUOTIENT",
```

Comment:
Could you please use `DeclareGlobalFunction` and `InstallGlobalFunction` for this function, as this function is not intended to be used in the same `.gi` file only?